pkg/server/container_create.go

```diff
@@ -91,8 +91,14 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	// Generate unique id and name for the container and reserve the name.
 	// Reserve the container name to avoid concurrent `CreateContainer` request creating
 	// the same container.
+	// TODO(teawater): make container create work using image name if metadata name
```

Comment:
1) I'm not sure whether we'd like to do that.
2) Even if we'd like to do that, we should do it in crictl instead of the cri plugin.